compiler-rt/lib/hwasan/scripts/hwasan_symbolize

```diff
@@ -336,16 +368,18 @@ class Symbolizer:
         size = local[4]
         if frame_offset is None or size is None:
           continue
-        obj_offset = (self.__last_access_address - fp - frame_offset) & fp_mask
-        if obj_offset >= size:
-          continue
+        obj_offset = (self.__last_access_address & fp_mask) - ((fp + frame_offset) & fp_mask)
```

Comment:
why? this is very biased towards overflow and against underflow. I.e., with fp_mask of 1Mb, access at (1Mb - 8 bytes) and the object offset of 8 bytes this code will come up with obj_offset of (1Mb - 16) instead of (-8).

Reply:
good point, thanks. I will rework this.

Reply:
PTAL.